### PR TITLE
Update modification drawer 

### PR DESCRIPTION
**Pull request by JacquesBorsenberger**
based on PR from @niconoir

### CODE REVIEW

File: src/components/graph/menus/network-modification-node-editor.js [hunk 1]
```diff
@@ -77,24 +82,26 @@ const NetworkModificationNodeEditor = ({ selectedNode }) => {
...
                 <Divider />
                 {modifications?.map((item) => (
                     <ModificationListItem key={item.uuid} modification={item} />
                 ))}
             </List>
 
-            <Button
+            <Fab
                 className={classes.addButton}
+                color="primary"
```
Comment (sBouzols) [inline src/components/graph/menus/network-modification-node-editor.js:100]: color should adapt from the dark or light theme, not sure you must set it here.

File: src/translations/network-modifications-en.json [hunk 2]
```diff
@@ -5,5 +5,6 @@
     "network_modifications/GENERATOR_CREATION": "Creating generator {computedLabel}",
     "network_modifications/LINE_CREATION": "Creating line {computedLabel}",
     "network_modifications/TWO_WINDINGS_TRANSFORMER_CREATION": "Creating 2 windings transformer {computedLabel}",
-    "network_modifications/BRANCH_STATUS": "Modification status line {computedLabel}"
+    "network_modifications/BRANCH_STATUS": "Modification status line {computedLabel}",
+    "network_modifications/SUBSTATION_CREATION": "Creation substation {computedLabel}"
```
Comment (sBouzols) [inline src/translations/network-modifications-en.json:9]: creating ? like for others CREATION trads ?

File: src/components/util/ask-text-dialog.js [hunk 3]
```diff
@@ -33,6 +33,10 @@ export const AskTextDialog = ({ title, value, show, onValidate, onClose }) => {
         setCurrentValue(e.target.value);
     };
 
+    useEffect(() => {
+        setCurrentValue(value);
```
Comment (sBouzols) [inline src/components/util/ask-text-dialog.js:37]: we mix controlled and uncontrolled value props component here... this is strange...
if value is passed as props, then we should pass a onChange callback as well then we coulad avoid this useEffect and the handleChange, and the surrentValue State.

**Review by sBouzols — COMMENTED** [review]
some comments for now...

File: src/translations/network-modifications-en.json (hunk 2 above)
Comment (JacquesBorsenberger) [reply src/translations/network-modifications-en.json]: done

File: src/components/util/ask-text-dialog.js (hunk 3 above)
Comment (JacquesBorsenberger) [reply src/components/util/ask-text-dialog.js:37]: we do not want to change title unless we validate the dialog (but when we change node, we want to modify the default value)

File: src/components/graph/menus/network-modification-node-editor.js (hunk 1 above)
Comment (JacquesBorsenberger) [reply src/components/graph/menus/network-modification-node-editor.js:114]: it does.
https://v4.mui.com/components/floating-action-button/

Comment (sBouzols) [reply src/components/graph/menus/network-modification-node-editor.js:114]: my bad

File: src/components/util/ask-text-dialog.js [hunk 4]
```diff
@@ -29,10 +29,19 @@ import PropTypes from 'prop-types';
 export const AskTextDialog = ({ title, value, show, onValidate, onClose }) => {
     const [currentValue, setCurrentValue] = useState(value);
     const intl = useIntl();
+
+    useEffect(() => {
```
Comment (etiennehomer) [inline src/components/util/ask-text-dialog.js:33]: We need both useEffect here ?

File: src/translations/network-modifications-fr.json [hunk 5]
```diff
@@ -3,6 +3,7 @@
     "network_modifications/EQUIPMENT_DELETION": "Suppression de {computedLabel}",
     "network_modifications/SUBSTATION_CREATION": "Création du site {computedLabel}",
     "network_modifications/VOLTAGE_LEVEL_CREATION": "Création du poste {computedLabel}",
+    "network_modifications/EQUIPMENT_ATTRIBUTE_MODIFICATION": "Equipment modification {computedLabel}",
```
Comment (etiennehomer) [inline src/translations/network-modifications-fr.json:6]: Modification de l'équipement

File: src/components/util/ask-text-dialog.js (hunk 4 above)
Comment (JacquesBorsenberger) [reply src/components/util/ask-text-dialog.js:33]: nope, just an auto merge with another, will remove the second one

File: src/translations/network-modifications-fr.json (hunk 5 above)
Comment (JacquesBorsenberger) [reply src/translations/network-modifications-fr.json]: done

**Review by etiennehomer — APPROVED** [review]

**Review by sBouzols — APPROVED** [review]
Code Review ok
No tests for now I will

**Comment by sBouzols** [comment]
Tests OK
All acceptances Criteria are OK in User Story

**Pull request merged**